packages/graphql/tests/tck/tck-test-files/schema/connections/unions.md

```diff
@@ -38,636 +38,607 @@ interface Wrote {
 
```

Comment:
Any chance you could format the schema TCK tests with the old `gql` tag trick so that these are easier to review?

Reply:
Yea